assets/index.html

```diff
@@ -29,7 +29,7 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script src="/config/admin-client-config.js"></script>
+    <script src="./config/admin-client-config.js"></script>
```

Comment:
This was intended to load the configuration from the shogun backend like this: `https://[domain]/config/admin-client-config.js`
I guess this won't work anymore as this should try to get the config from `https://[domain]/admin/config/admin-client-config.js`.

Reply:
Yes, I believe this assumption exists from times where the admin client was intended to be shipped from SHOGun. As we changed this to an image based approach, I think it's much clearer to expect this file to be included in this image rather than the SHOGun image/deployment.